doc/webdashboard/Web-Dashboard.md

```diff
@@ -1,123 +1,24 @@
-* Run your instrument
-* Run JSON Bourne up as above
-
-If you need to update the archive engine then you will need to:
-
-1. Run create_icp_binaries.bat
-1. `make clean uninstall install` in `..\EPICS\CSS\master`
+See {ref}`pvws`
 
 {#webdashboard_troubleshooting}
 ## Troubleshooting
 
-### General Investigation
-
-First look at the log to ensure that there are no issues. The log is held in `C:\JSON_Bourne\log`. Issues may be in the front end, in which case error logs are in the web browser, visit the webpage in a browser and open up the web console.
-If there are a number of `HTTP Error 503. The service is unavailable` errors, restarting the server completely may be required, but simply restarting the Dataweb should be the first thing to try.
-
-{#webdashboard_restart_dataweb}
-### Restart the Dataweb
-
-As admin open the "Task Scheduler" (there is a shortcut for this on the desktop) and end and run the "JSON Bourne" task (in task scheduler library). Make sure that ending the task has killed the Python webserver process.
-
-### New Instrument with No Details
-
-If the instrument archive has never been restarted then the dataweb will fail to show any information and claim that the server hasn't been started. To fix this simple restart the instrument archive.
-
-{#webdashboard_troubleshooting_instrument_page_not_working}
-### Instrument Page not Working on Web Dashboard
-
-Several causes
-
-1. Check that the instrument is in the list of Instruments in https://github.com/ISISComputingGroup/JSON_bourne/blob/master/webserver.py and that the version on web server is up-to-date.
-
-1. Issues with MySQL in the moment the IBEX server was started (this seems to affect the archiver start up). Check logs of the MySQL service in the `var` area, fix any issues so that MySQL is running correctly again, then restart the IBEX server.
-
-1. If it works in your browser but not he users they may have a old cached copy (this shouldn't happen but we have seen it in Safari). Clear their browser cache and reload.
-
-1. Try restarting `ARINST` on the instrument. It can happen that the archiver does not pick up all PVs to be archived on server startup. A symptom of this is that the configuration file under `EPICS\CSS\master\ArchiveEngine\inst_config.xml` is very short compared to other machines.
-
-## Future Development Ideas
-
-* We need to improve the unit test coverage of this project. It would be worth looking into the [requests-mock](https://pypi.python.org/pypi/requests-mock) library as this would make it very easy to test server code which makes HTTP requests.
-
-## Overview page
+### Web dashboard showing outdated or incorrect information
 
-http://dataweb.isis.rl.ac.uk/IbexDataweb/Overview/ibexOverview.html
\ No newline at end of file
+If you cannot subscribe directly to a PV by using the [PVWS homepage](https://ndaextweb4/pvws/) ie it gets no update with a value in it, this is likely due to a central gateway issue or an issue with the NDX.
```

Comment:
Can we clarify "central gateway running on control-svcs, or the external gateway on the NDX"? 

And maybe give some troubleshooting steps like:
- On EXTWEB point a caget at EXTWEB's restricted gateway to prove whether issue is EPICS level or PVWS level
- On EXTWEB point a caget at central gateway to prove whether issue is at that end
- Point caget at an individual instrument, bypassing all gateways other than external gateway on each instrument